src/Templating/test/Templates.Test/Helpers/TemplateTestBase.cs

```diff
@@ -32,7 +31,7 @@ public TemplateTestBase(ITestOutputHelper output)
             _output.Value = output;
             TemplatePackageInstaller.EnsureTemplatingEngineInitialized(output);
 
-            ProjectGuid = Guid.NewGuid().ToString("N");
+            ProjectGuid = Guid.NewGuid().ToString("N").Substring(0, 6);
```

Comment:
I was running into Windows path length limitations. The RazorComponents template has an extra level of folder nesting (because it has a `.sln` at the top level, then two `.csproj` templates in subfolders), so it was necessary to make the project names short enough that it doesn't all explode.